Source/UnrealEnginePython/Private/UnrealEnginePython.cpp

```diff
@@ -113,7 +113,12 @@ void FUnrealEnginePythonModule::RunString(char *str) {
 
 void FUnrealEnginePythonModule::RunFile(char *filename) {
 	FScopePythonGIL gil;
-	char *full_path = TCHAR_TO_UTF8(*FPaths::Combine(*FPaths::GameContentDir(), UTF8_TO_TCHAR("Scripts"), *FString("/"), UTF8_TO_TCHAR(filename)));
+	char *full_path = (char *)malloc(strlen(filename) + 1);
```

Comment:
here i think a single pointer copy is enough:

full_path = filename

nevertheless remember to free() malloc'ed memory